test/unit-test/ota_config.h

```diff
@@ -31,6 +31,28 @@
 #ifndef OTA_CONFIG_H_
 #define OTA_CONFIG_H_
 
-/* Dummy ota_config.h for building the library. */
+#define configENABLED_DATA_PROTOCOLS    ( OTA_DATA_OVER_MQTT | OTA_DATA_OVER_HTTP )
+
+#define LOG_LEVEL_ERROR                 0
+#define LOG_LEVEL_WARN                  1
+#define LOG_LEVEL_INFO                  2
+#define LOG_LEVEL_DEBUG                 3
+
+#ifndef LOG_LEVEL
+    #define LOG_LEVEL                   -1
```

Comment:
Shouldn't the default value for log be LOG_LEVEL_ERROR?

Reply:
I find it bit noisy to turn on log when running unit tests since many are testing failing cases. I added this only for checking compiler warnings.